.github/SUPPORT.md

```diff
@@ -8,6 +8,6 @@ If you know how to fix the issue, feel free to send a pull request our way. (The
 [Official Support Policy]: https://docs.microsoft.com/powershell/scripting/powershell-support-lifecycle
 [FAQ]: https://github.com/PowerShell/PowerShell/tree/master/docs/FAQ.md
 [Contribution Guides]: https://github.com/PowerShell/PowerShell/tree/master/.github/CONTRIBUTING.md
-[known issues]: https://docs.microsoft.com/powershell/scripting/whats-new/known-issues-ps6
+[known issues]: https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell
```

Comment:
Where's this supposed to point?

Reply:
Same place as the rest. I didn't want to reword the "known issues" part of the paragraph. It was descriptive enough for the purpose of the message. Inserting the doc title would be more jarring and not add value.